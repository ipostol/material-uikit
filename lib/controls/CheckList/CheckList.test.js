'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _enzyme = require('enzyme');

var _Checkbox = require('../Checkbox');

var _Checkbox2 = _interopRequireDefault(_Checkbox);

var _CheckList = require('./CheckList');

var _CheckList2 = _interopRequireDefault(_CheckList);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

describe('CheckList component', function () {

  it('should render correctly', function () {

    var props = {
      options: [{ id: 1 }],
      value: [],
      onChange: function onChange() {}
    };

    var wrapper = (0, _enzyme.shallow)(_react2.default.createElement(_CheckList2.default, props));

    wrapper.find(_Checkbox2.default).simulate('change');

    expect(wrapper).toMatchSnapshot();
  });
});