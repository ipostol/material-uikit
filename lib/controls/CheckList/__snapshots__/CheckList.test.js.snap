// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckList component should render correctly 1`] = `
<div
  className="container"
>
  <Checkbox
    checked={false}
    className="label"
    onChange={[Function]}
  />
</div>
`;
